python/ray/train/examples/mlflow_fashion_mnist_example.py

```diff
@@ -1,7 +1,7 @@
 import argparse
 
 from ray.air.config import RunConfig, ScalingConfig
-from ray.train.examples.torch_fashion_mnist_example import train_func
+from ray.train.examples.pytorch.torch_fashion_mnist_example import train_func
 from ray.train.torch import TorchTrainer
 from ray.air.callbacks.mlflow import MLflowLoggerCallback
```

Comment:
should this example be moved into `pytorch`?

Reply:
I guess it is using pytorch, but on the other hand, it's supposed to show off MLFlow first and foremost and pytorch is only an implementation detail here. I'd just leave it where it is for better visibility.